db/migrate/20201217105811_add_ga_user_count_to_project.rb

```diff
@@ -0,0 +1,6 @@
+class AddGaUserCountToProject < ActiveRecord::Migration[5.0]
+  def change
+    add_column :projects, :google_analytics_user_count, :integer
+    add_column :projects, :ga_view_id, :string
```

Comment:
I think you want `products` here

Reply:
I couldn't actually run it myself with it renamed to `product`, because the renaming of Project to Product actually occurs chronologically after this migration as that PR was merged in before this one. Hence I had to rename it to Project to get my DB working (from scratch)

Reply:
Oh I ssee because the date of this migration is 2020. In which case, I think the migration date in the filename should be updated to Jun 2021 and then it should be ok 🤞

Reply:
That would work too. Technically a bit of a hack, but I can do it